src/dbnode/persist/fs/seek_manager.go

```diff
@@ -218,11 +232,19 @@ func (m *seekerManager) CacheShardIndices(shards []uint32) error {
 		byTime.accessed = true
 		byTime.Unlock()
 
-		if err := m.openAnyUnopenSeekersFn(byTime); err != nil {
-			multiErr = multiErr.Add(err)
-		}
+		wg.Add(1)
+		workers.Go(func() {
+			defer wg.Done()
```

Comment:
It should be safe to put this after the `openAnyUnopenSeekersFn` call and drop the `defer`, looks like there's no alternate path here

Reply:
Sg